src/components/CreateProject/index.tsx

```diff
@@ -0,0 +1,190 @@
+import React, { useEffect, useState } from 'react';
+import Breadcrumb from 'components/Breadcrumb';
+import { Project, ProjectStatus, Role, User } from 'utils/types';
+import { USERS } from 'fakeData/users';
+import './index.scss';
+import { useCreateProjectForm } from './useCreateProjectForm';
+import { useHistory } from 'react-router-dom';
+import AddMember from './AddMember';
+
+function CreateProject() {
+   const history = useHistory();
+   const [listUsers, setListUsers] = useState<User[]>([...USERS]);
+   const [listUsersInProject, setListUsersInProject] = useState<User[]>([]);
+
```

Comment:
như comment ở trên. allUsers là tất cả các member (chỉ là Member - k phải cả PM với Admin)

```suggestion
   const allMemberUsers = useMemo(() => {
         return USERS.filter(...........)
   }, [])

```